doc/AzurePipelines.md

```diff
@@ -0,0 +1,78 @@
+# Azure Pipelines
+
+## Pipeline Code
+This project uses CI/CD pipelines that are implemented as yaml code.
+They are declared in the following files.
+- `.azure-pipelines.yml`
+
+These pipelines are divided in parameterized stages that are defined accross several files, all located under [`build/`](build/).
+The more complex stages are also divided into several steps files, again all located under the build folder.
+
+## Variable Groups and Secrets
+These pipelines rely on a few variable groups and secrets in order to fully work. These are documented in [`variables.yml`](build/variables.yml).
+
+## Main Pipeline [azure-pipelines.yml](../build/azure-pipelines.yml)
+
+At high level, the CI/CD pipelines do the following:
+- **Build** the app in **staging**.
+  - **Deploy** the staging app (to AppCenter and/or TestFlight and GooglePlay).
+- **Build** the app in **production**.
+  - **Deploy** the production app (to TestFlight and GooglePlay).
+
+It also runs automated tests during the build steps.
+
+### Pull request runs
+Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**. To reduce the build time, some runtime performance optimizations are disabled for PR builds. This requires a specific variable called `IsLightBuild` to be set, hence why it is appearing in the pipeline. 
```

Comment:
That doesn't seem true. The `IsLightBuild` variable seems to only be used to disable some stages and some steps.

Reply:
Well I feel like it is. Could probably rephrase it, but we are actually not archiving and signing the IPA on iOS when it's a PR build.

Reply:
> To reduce the build time, some runtime performance optimizations are disabled for PR builds

That's not true.